### DISCUSSION TITLE
email-domains and emails-outside

### DISCUSSION THREAD

**Discussion by chipmanfu** [cmu-sei/GHOSTS#115] [General]
I've configured the emails-outside.json list, however ghost isn't sending emails to any of the email addresses listed in emails-outside.json.  In order to make ghost send both internal and external emails, I've added external emails to the emails-domain.json.  However, it would be nice to keep these lists separated.  Is there a configuration option I'm missing so that Ghost will randomly send emails from the emails-outside.json list?

**Comment by sei-dupdyke** [comment]
Sorry it took me a bit to get to this. It seems to work for me, but there are a couple of settings to review in `config/application.json`:

"RecipientsOutsideMin": 0,
"RecipientsOutsideMax": 1,

Change the min value to 1 and see if that corrects things please.

**Comment by sei-dupdyke** [comment]
Aha, I found the bug and fixed it in patch 6.1.8.0. Sorry for the confusion.
